src/components/dummy

```diff
@@ -0,0 +1,75 @@
+login first code
+
+<script>
+export default {
+  name:'Register',
+  data(){
+    return{
+      name:'',
+      email:'',
+      password:''
+    }
+  },
+  methods:{
+    register(){
+      console.log(this.name)
+      this.$store.dispatch('REGISTER',{
+        name:this.name,
+        email:this.email,
+        password:this.password
+      })
+      .then(response=>{
+        console.log("Register.vue Response"+response)
+        this.$router.push('/Login')
+      })
+      .catch(error=>{
+        console.log("Registration.vue error"+error)
+      })
+    }
+  }
+
+}
+</script>
+
+<style>
+
+</style>
+
+
```

Comment:
Add this to its place and delete dummy component, For backup comment existing code with comments
@hrithik20